modules/installation-registry-storage-config.adoc

```diff
@@ -8,10 +8,35 @@
 // * installing/installing_vsphere/installing-vsphere.adoc
 // * installing/installing_ibm_z/installing-ibm-z.adoc
 
+ifeval::["{context}" == "installing-aws-user-infra"]
+:aws:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-aws"]
+:aws:
+endif::[]
+
 [id="installation-registry-storage-config_{context}"]
 = Image registry storage configuration
 
-If the `image-registry` Operator is not available, you must configure storage
-for it. Instructions for both configuring a PersistentVolume, which is required
+ifdef::aws[]
+Amazon Web Services provides default storage, which means the `image-registry`
+Operator is available after installation. However, if the Registry Operator
+cannot create an S3 bucket, and automatically configure storage, you must
```

Comment:
Possibly drop comma after bucket?

Reply:
Yeah, a little choppy there. I'll remove it.